### ISSUE TITLE
service request notifications

### ISSUE THREAD

**Issue by JChipault** [USGS-WiM/whispersservices_django#342]
Here's the long message for service requests:
User [C] with organization [F] has requested [O] services for event [A] in [P]. 

Comments: [L2]

1)  can we just use "Diagnostic" or "Consultative" for the service requests for variable O?

2) looks like we don't need the "Comments: [L2]" part of this message because even comments submitted in the same popup as the initial service request are being sent out in a separate "Service Request Comments" notification. Alternatively, any way that the initial comment could come through with the service request (with any additional comments coming through on the "service request comments")?

**Comment by aaronstephenson** [comment]
For 1, you can change the name values in the servicerequesttype table to whatever you like. I am not referencing those values anywhere in my code (not sure if the front end is, tho).

For 2, removing the Comments part of the service request notification is easy. The alternative would require some work.

**Comment by aaronstephenson** [comment]
After discussion IRL at the biweekly team meeting, I will include the initial comment in the Service Request notification, and not send a separate Comment notification. Any subsequent comments will still trigger a Comment notification as expected.

**Comment by aaronstephenson** [comment]
Implemented in 00b969e0

**Comment by aaronstephenson** [comment]
@JChipault in the test environment, any new service requests with a comment will not include that initial comment in the notification and email.

**Comment by JChipault** [comment]
How hard would it be to separate out the lines in the email

![image](https://user-images.githubusercontent.com/49287076/77774724-87058280-7019-11ea-9b56-7f5063cf0b82.png)

if there are two comments entered with the initial request subform?

![image](https://user-images.githubusercontent.com/49287076/77774805-a2708d80-7019-11ea-8d16-2314ef82b0aa.png)

If this is a pain, don't worry about it

**Comment by JChipault** [comment]
![image](https://user-images.githubusercontent.com/49287076/77778727-722bed80-701f-11ea-87be-7426591bc6f4.png)

The first two comments above were bundled up with the service request email 

So there's only one other comment that I should've gotten an email about, but I got three comment notifications:

![image](https://user-images.githubusercontent.com/49287076/77779805-0f3b5600-7021-11ea-95c4-0cc84388c8fd.png)

**Comment by BlakeDraper** [comment]
@aaronstephenson FYI 👆

**Comment by aaronstephenson** [comment]
Already working on this.

**Comment by BlakeDraper** [comment]
Sorry, I forgot this was already in your repo. Thought I was on a different one. Jumping between a bunch of issues.

**Comment by aaronstephenson** [comment]
@JChipault the two comments are already split across two lines. Here's the literal value from the notification table:

```
<p>User Jenny Chipault with organization U.S. Geological Survey - National Wildlife Health Center has requested Diagnostic and Consultative Services services for event <a href="https://whisperstest.wim.usgs.gov/whispers/event/170864">170864</a> in Merced, CA, USA.</p>

<p>Comments: 
TEST: new request with associated comment
what happens if I do a sub comment in the same subform?</p>
```

I suspect the problem is that the email isn't rendering `\r\n`, so I'll look into possibly swapping that with `<br />`.

**Comment by aaronstephenson** [comment]
Fixed in 645c1a74
